ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/BackupReplayFile.java

```diff
@@ -107,9 +140,33 @@ public void write(final ConsumerRecord<byte[], byte[]> record) throws IOExceptio
   }
 
   @Override
-  public void close() throws IOException {
-    if (writer != null) {
-      writer.close();
+  public void close() {
+  }
+
+  private void initDirtyCopy() throws IOException {
+    final Path dirtyFile = dirty(file).toPath();
+    Files.deleteIfExists(dirtyFile);
+    Files.deleteIfExists(tmp(file).toPath());
+    if (file.exists()) {
```

Comment:
Would we want to throw here if the file doesn't actually exist or is it okay if it doesn't exist? It doesn't seem like we'd note that we didn't do anything here

Reply:
It's ok for it not to exist - that's the case for new clusters. That said, I think we need to do some initialization in that case - not sure how this worked.

Reply:
@rodesai just confirming - is that initialization something we need to add or is this good to go?

Reply:
> That said, I think we need to do some initialization in that case - not sure how this worked.

Not sure I understand this sentence here: if it's a new cluster, then it's okay to have all three: live copy, dirty copy, and tmp, to be empty at initialization, since when the first append happens we would lazily created those files then, right?